src/connection/WalletConnectV2.ts

```diff
@@ -16,7 +16,7 @@ const RPC_URLS_WITHOUT_FALLBACKS = Object.entries(RPC_URLS).reduce(
 )
 const optionalChains = [...L1_CHAIN_IDS, ...L2_CHAIN_IDS].filter((x) => x !== SupportedChainId.MAINNET)
 
-export class WalletConnectV2Popup extends WalletConnect {
```

Comment:
nit: just `WalletConnectV2` to match the filename. `Analytics` is an implementation detail.